ConsoleApp1/Program.cs

```diff
@@ -4,6 +4,63 @@ internal class Program
 	{
 		static void Main(string[] args)
 		{
+			string str = null;
+			string str2 = "";
+			string str3 = "12345A";
+			string str4 = "12345A";
+			string str5 = "123";
+			string str6 = "qwertyuiopasdfghjklzxsecvbm";
+			string str7 = "1asw34#@8765^";
+			int ints = 0;
+			int ints2 = 0;
+			Console.WriteLine(Compare(str3,str4));
+			Analyze(str7);
+			Console.WriteLine(Sort(str6));
+			//Duplicate(str4,str5);
```

Comment:
As for duplicates, you should compare each character with the next characters, if one is equals with another then it's duplicate
OR
find first and last index of each character if they are different then they are duplicates